include/lbann/layers/regularizers/batch_normalization.hpp

```diff
@@ -363,7 +363,9 @@ class batch_normalization_layer : public regularizer_layer {
     assert_eq(m_mean->Matrix().Width() * m_mean->Matrix().Height(),
               num_channels);
 
-    dc::Shape per_channel_stat_shape({1, 1, num_channels, 1});
+    std::vector<int> per_channel_stat_shape_v(dc::num_dims, 1);
```

Comment:
This can be simply written as:
```
dc::Shape per_channel_stat_shape(dc::num_dims, 1);
per_channel_stat_shape[dc::num_spatial_dims] = num_channels;
```